### DISCUSSION TITLE
Displaying images

### DISCUSSION THREAD

**Discussion by itaysharir** [FelixKratz/SketchyBar#258] [Q&A] [answered]
How can i add an item that displays an image to the bar?

I tried this:
sketchybar      --add       item               discord right                                                                                           \
sketchybar       --set       discord           image=/Users/itaysharir/.config/sketchybar/discord-logo.png          \

I know there is probably something really easy and stupid that I'm not getting.
Thanks in advance!

**Comment by FelixKratz** [comment, marked as answer]
Images are available as properties of a `background`. Thus you want to do this:
```bash
sketchybar --set <name> background.image=<path>
```

**Comment by itaysharir** [comment]
sketchybar --add item discord right \ 
sketchybar --set discord background.image=/Users/itaysharir/.config/sketchybar/discord-logo.png \

Doesn't work. what am i doing wrong?

**Reply by FelixKratz** [reply]
You need to enable the background:
```bash
sketchybar --set <name> background.drawing=on
```

**Reply by itaysharir** [reply]
Thanks! Didn't realize the background wasn't enabled by default.

**Reply by FelixKratz** [reply]
I might rethink the logic for automatic background activation, since I see that it can be confusing the way it is currently.
